### ISSUE TITLE
Lamp State Stuck on one Value

### ISSUE THREAD

**Issue by N1kless** [bezmi/hass_custom_components#27]
First of all thank you for this integration, it works very well and gives me the opportunity to make the control of my little cinema much easier especially for the family. However the attribute lamp_state is stuck on the high value all the time. I would like to have a lamp low/high indicator in my HA Dashboard. I own a JVC N5. Everything else seems to work as expected I can change lamp mode, Lens memory and get the other state information. 

Any Idea if this is a Bug or If I can influence this somehow?

TLDR: Lamp_state is always high on JVC N5 and does not change. What can I do?
